src/components/CollapsibleCard/CollapsibleCard.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+// Components
+import Card from '../Card/Card';
+
+// Style
+import style from './style.scss';
+
+const styleColors = { RED: 'red' };
+
+class CollapsibleCard extends Component {
+  state = {
+    displayCard: false,
```

Comment:
let's use `shouldDisplayCard` since that's a convention for booleans (`displayCard` is a verb and suggests that it's a method that displays a card)

Reply:
do we want collapsibility to be controlled by props too? e.g. if the component is connected (in that case we'd need to support passing it via props)

Reply:
We talked about this earlier. I added a prop to determine if the collapsible element is displayed on first render.